README.md

```diff
@@ -364,6 +364,9 @@ We've also provided a generator which creates a skeleton job and updates the ini
 bin/rails g shopify_app:add_after_authenticate_job
 ```
 
+If you want to perform that action only once, e.g send a welcome email to the user when he installs the app, you should make sure that this action is idempotent, meaning that it won't have an impact if run multiple times.
```

Comment:
e.g send a welcome email to the user when ~he installs the app~ they install the app